string-constants-doc/string-constants/translation-guide.scrbl

```diff
@@ -0,0 +1,161 @@
+…
+          (prefix-in yourlanguage: "private/yourlanguage-string-constants.rkt"))
+]
+
+and
+@racketblock[
+(define built-in-string-constant-sets
+  (list
+   (make-sc 'english english:string-constants #f)
+…
+   (make-sc ' yourlanguage yourlanguage:string-constants #f)))
+]
+
+And with that you are done with the initial work — you have created
+a template for your language (still filled in with the English
+originals) and you have told Racket to use it. We are off to the races
+and all we need to do is fill in the translations.
+
+
+@; ----------------------------------------------------------------------
+
+@section{Translation}
+
+Open @tt{yourlanguage-string-constants.rkt} in your favorite text
+editor. It has a fairly trivial structure: it is basically a map — a
+long list of key-value pairs. The key is the identifier of the string
+— how Racket source code is going to refer to the message. Do not
+change these. Value is the translation. As you started with the
+English version — you have all the comments and the formatting. I
+highly recommend keeping these in place because they are very helpful
+when you need to update the translations.
+
+The first thing you need to do is fix the line:
+@racketblock[(module english-string-constants "string-constant-lang.rkt"
+…)]
+
+It has to become
+@racketblock[(module yourlanguage-string-constants "string-constant-lang.rkt"
+…)]
+
+
+The second thing is to add some metadata to your file. You can use
+standard comments in the file – either prefix a line with double
+semicolon @tt{;;} or comment out a block of text via @tt{#| commented
+block |#}.
+
+In that part of the file put some basic data — like your name, which
+version of the @tt{private/english-string-constants.rkt} file you are
+syncing with. This is also the place to put out conventions for the
+translation such as specific terms, style etc. I would also recommend
+spelling out the license of it. For example in the
+@link["https://github.com/racket/string-constants/blob/master/string-constants-lib/string-constants/private/bulgarian-string-constants.rkt"]{Bulgarian
+translation} I have explicitly pointed out:
+
+@tt{
+;; This file is distributed under the same terms as Racket
+}
+
+to ease with relicensing. Currently translations are double licensed
+— under the
+@link["https://github.com/racket/string-constants/blob/master/LICENSE"]{Apache
+2.0 license and the MIT license}.
+
+Now all you need is 3 afternoons to go over and translate the strings.
+
+All the translated strings are quoted by double quotes. So if you need
+to put @tt{"} in the translation you would need to quote it like
+@tt{\"}. However I recommend you use
+@link["https://en.wikipedia.org/wiki/Quotation_mark#Summary_table"]{proper
+quotation marks for your language}.
+
+A new line is represented by @tt{\n}.  @tt{~a} is substituted with
+whatever parameter is given to the string template. And that is
+basically it.
```

Comment:
Cut "And that is basically it", I'd say.

Reply:
Done